src/read.c

```diff
@@ -6019,12 +6037,9 @@ avifResult avifDecoderReset(avifDecoder * decoder)
                     avifDiagnosticsPrintf(data->diag, "Box[sato] input item %u is not a supported image type", inputImageItem->id);
                     return AVIF_RESULT_DECODE_SAMPLE_TRANSFORM_FAILED;
                 }
-                // Input image item order is important because input image items are indexed according to this order.
-                AVIF_CHECKERR(inputImageItem->dimgIdx == data->sampleTransformNumInputImageItems, AVIF_RESULT_NOT_IMPLEMENTED);
```

Comment:
This is no longer checked?

Reply:
I missed the `avifItemCategory * category = &data->sampleTransformInputImageItems[inputImageItem->dimgIdx];` part, thanks!

Reply:
> I don't see why this was needed in the first place

Just to make sure items were in the same order are their `dimg` index, meaning not having to bother thinking how to implement it otherwise.